ftp-source/pom.xml

```diff
@@ -0,0 +1,58 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>spring-cloud-stream-modules</artifactId>
+		<groupId>org.springframework.cloud.stream.module</groupId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>ftp-source</artifactId>
+	<packaging>jar</packaging>
+	<name>ftp-source</name>
+	<description>FTP Source stream module</description>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<start-class>org.springframework.cloud.stream.module.log.FtpSourceApplication</start-class>
+	</properties>
```

Comment:
I've been bitten wy this as well (somehow IDEA changing more text than it should).
But, why are we even specifying this? If I reckon correctly, the boot plugin can find the main class itself, can't it? Assuming it fails on ambiguity, this may be the best thing to do, wdyt?